BUILD.bazel

```diff
@@ -267,96 +26,124 @@ cc_library(
 # )
 ################################################################################
 
-proto_library(
+alias(
     name = "any_proto",
-    srcs = ["src/google/protobuf/any.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:any_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "api_proto",
-    srcs = ["src/google/protobuf/api.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:api_proto",
     visibility = ["//visibility:public"],
-    deps = [
-        "//:source_context_proto",
-        "//:type_proto",
-    ],
 )
 
-proto_library(
+alias(
     name = "duration_proto",
-    srcs = ["//:src/google/protobuf/duration.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:duration_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "empty_proto",
-    srcs = ["src/google/protobuf/empty.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:empty_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "field_mask_proto",
-    srcs = ["src/google/protobuf/field_mask.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:field_mask_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "source_context_proto",
-    srcs = ["src/google/protobuf/source_context.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:source_context_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "struct_proto",
-    srcs = ["src/google/protobuf/struct.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:struct_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "timestamp_proto",
-    srcs = ["src/google/protobuf/timestamp.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:timestamp_proto",
     visibility = ["//visibility:public"],
 )
 
-proto_library(
+alias(
     name = "type_proto",
-    srcs = ["src/google/protobuf/type.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:type_proto",
     visibility = ["//visibility:public"],
-    deps = [
-        "//:any_proto",
-        "//:source_context_proto",
-    ],
 )
 
-proto_library(
+alias(
     name = "wrappers_proto",
-    srcs = ["src/google/protobuf/wrappers.proto"],
-    strip_import_prefix = "src",
+    actual = "//src/google/protobuf:wrappers_proto",
     visibility = ["//visibility:public"],
 )
 
-# Built-in runtime types
+# Source files: these are aliases to a filegroup, not a `proto_library`.
+#
+# (This is _probably_ not what you want.)
```

Comment:
Is this in place just for compatibility? Do we have specific users that we know of? Can we remove this once we burn them down?

Reply:
For the old name `lite_well_known_protos`, it's compatibility and that name can probably go away soon-ish. But I think we will probably need to provide access to the sources for the foreseeable future (so `well_known_type_protos` probably ought to stick around), and I sort of think that the root package is the right place to put it. But that can be a future change.